phpunit/functional/Glpi/Inventory/InventoryTest.php

```diff
@@ -8725,10 +8725,9 @@ function createJsonString($versionClient)
         $nb_computers = countElementsInTable(\Computer::getTable());
 
         // Second inventory
-        $json = json_decode(createJsonString("GLPI-Agent_v1.10-dev2"));
```

Comment:
You must build a new json object, the existing one is currently modified by the inventory process (even if it should not, I did not take time to investigate since it's not a big deal).

But we'll talk about that tomorrow in our internal channel ;)